### ISSUE TITLE
Matrix-bridge for Telegram

### ISSUE THREAD

**Issue by Sir-Photch** [MM2-0/Kvaesitso#279]
I've seen that there is a Telegram channel for Kvaesitso, and I'd really like to join.
However, I despise Telegram in favor of a subjectively better alternative: Matrix!
Maybe there are also other users out there that prefer matrix for communication.

Fortunately, it is possible to bridge Telegram into Matrix and vice-versa:
https://t2bot.io/telegram/

Setup seems quite simple.
This would need a Kvaesitso matrix room, of course.
As a first iteration, one at the matrix.org-homeserver should be sufficient.

**Comment by daywalk3r666** [comment]
Little info. If max set up a matrix room. The room must be unencrypted

**Comment by Sir-Photch** [comment]
I don't think this is necessarily the case, but even then, what is the point of encrypting a public room, where anyone can join?

**Comment by MM2-0** [comment]
> Maybe there are also other users out there that prefer matrix for communication.

I created a poll: https://github.com/MM2-0/Kvaesitso/discussions/283

Personally, I prefer Telegram, because it feels more mature and more user friendly than Element (and as far as I'm aware, there isn't any other feature complete client for Android). Also, Telegram probably has the higher user base. But let's see, if people show interest, I might create a Matrix space as well.
